src/v2/view-builder/utils/AuthenticatorUtil.js

```diff
@@ -24,11 +24,15 @@ const getButtonDataSeAttr = function(authenticator) {
   return '';
 };
 
-/* eslint complexity: [0, 0], max-statements: [2, 24] */
+/* eslint complexity: [0, 0], max-statements: [2, 25] */
 const getAuthenticatorData = function(authenticator, isVerifyAuthenticator) {
   const authenticatorKey = authenticator.authenticatorKey;
   const key = _.isString(authenticatorKey) ? authenticatorKey.toLowerCase() : '';
   let authenticatorData = {};
+  let nicknameText = isVerifyAuthenticator ? authenticator.relatesTo?.nickname : undefined;
+  if (nicknameText && nicknameText.length > 20) {
+    nicknameText = nicknameText.substring(0, 20) + '...';
+  }
```

Comment:
This should no longer be necessary right? Since you created the css in `assets/sass/v2/_select-authenticator.scss`

Reply:
correct - will remove

Reply:
Removed.